lib/src/core/util/genre_by_id_list_generator.dart

```diff
@@ -0,0 +1,9 @@
+import '../../domain/entity/genre.dart';
+
+class GenreListGenerator {
+  GenreListGenerator();
+
+  List<Genre> getGenresFromId(List<int> ids, List<Genre> genres) {
```

Comment:
trailing comma

Reply:
This could be made as a extension function that extends from List<Int> and returns a List<Genre>